### ISSUE TITLE
Cura Error: Unable to Slice

### ISSUE THREAD

**Issue by BMOtheConsole** [GioPalusa/adventurer3Cura#1]
Importing your Settings did go like described, but in Cura i cant Slice the Model.

The Error Message says "Unable to Slice" With no extra Information Why.

**Comment by GioPalusa** [comment]
Yeah. I’m trying to figure out why this is happening... I haven’t had time to look into it, but I will have more time the upcoming days to solve that issue...

**Comment by totisso** [comment]
Sorry buddy for the inconvenience but it's true. Unfortunately your settings doesn't work. Many people are waiting for your help.. We are on Cura 4.7.0..

**Comment by ajzbern** [comment]
profile is still unable to slice, whats going on with the fix? any progress or attempts

**Comment by Slade2000** [comment]
Yup same issue still. Can't Slice.

**Comment by KeltE** [comment]
Hello!
Can anyone test my branch from the Adventure 3 Definition. I don't have Adventure 3 but I made some changes based on the definition of a working Dreamer NX. Cura 4.8 at least slices now.
Egon
https://github.com/KeltE/adventurer3Cura

**Comment by camyrland** [comment]
> Hello!
> Can anyone test my branch from the Adventure 3 Definition. I don't have Adventure 3 but I made some changes based on the definition of a working Dreamer NX. Cura 4.8 at least slices now.
> Egon
> https://github.com/KeltE/adventurer3Cura

I tried it. Crashed my Cura, had to roll back and lost my current printer setup. 4.8.0

**Comment by KeltE** [comment]
Strange. I also tested Cura 4.8. I slice without any problems. I just checked again too, it works. 
![image](https://user-images.githubusercontent.com/66703581/113489106-d1da9580-94ca-11eb-9067-5f8651f7d207.png)

**Comment by KeltE** [comment]
He has not even noticed that the location in the teaching is completely wrong. Must copy on windows C: \ Program Files \ Ultimaker Cura 4.8.0
Did not view the read.me file at all. 
Sorry for the inconvenience.

**Comment by camyrland** [comment]
> Strange. I also tested Cura 4.8. I slice without any problems. I just checked again too, it works.
> ![image](https://user-images.githubusercontent.com/66703581/113489106-d1da9580-94ca-11eb-9067-5f8651f7d207.png)

I'm using Cura as a snap on Ubuntu, but that shouldn't matter..

**Comment by camyrland** [comment]
> He has not even noticed that the location in the teaching is completely wrong. Must copy on windows C: \ Program Files \ Ultimaker Cura 4.8.0
> Did not view the read.me file at all.
> Sorry for the inconvenience.

I tried again after setting up Cura again, and now it seems to work. Slices just nicely. I have the Adventurer 3 at work, so I won't get around to testing it until late next week.

**Comment by KeltE** [comment]
II discovered the profile from which branch I made is the tutorials completely wrong. And the original profile is almost a copy of Euger Creator pro's profile. Which is a completely different printer. Creator pro Sailfish commands are different. The location on Windows must be "C: \ Program Files \ Ultimaker Cura 4.8.0 \ resources" Not as specified in the original profile. I'm fixing my read.me file.
The printer appears under Flashforge3. Since I have a Flashforge manufacturer named Cura 4.9 comes with the included Dreamer NX printer. And otherwise, the separately copied printer does not appear in the list.

**Comment by camyrland** [comment]
> He has not even noticed that the location in the teaching is completely wrong. Must copy on windows C: \ Program Files \ Ultimaker Cura 4.8.0
> Did not view the read.me file at all.
> Sorry for the inconvenience.

For anyone interested, I've started a new thread over at KeltE's project: https://github.com/KeltE/adventurer3Cura/discussions/2

**Comment by totisso** [comment]
Does anybody knows where can I find the correct path on IOS? I’m trying to find out without success..

Thanks a lot..!!

**Comment by totisso** [comment]
Correction: 	is not IOS.. I meant macOS.. Sorry guys..

**Comment by KeltE** [comment]
Maybe helps. Found this tutorial.
[https://help.prusa3d.com/ja/guide/how-to-import-profiles-to-cura-4-x-windows-macos_17848](url)

**Comment by Razor412** [comment]
> Correction: is not IOS.. I meant macOS.. Sorry guys..
> […](#)

Did you ever find the correct location?
